ledger/sandbox-classic/src/main/scala/platform/sandbox/SandboxServer.scala

```diff
@@ -143,23 +145,23 @@ final class SandboxServer(
 
   private[this] val engine = {
     val engineConfig = {
-      val baseConfig =
-        if (config.seeding.isEmpty) {
-          if (config.devMode) {
+      val allowedLanguageVersions =
+        config.engineMode match {
+          case EngineMode.Stable if config.seeding.nonEmpty =>
+            LanguageVersion.StableVersions
+          case EngineMode.Stable =>
+            LanguageVersion.LegacyVersions
+          case EngineMode.EarlyAccess if config.seeding.nonEmpty =>
```

Comment:
Is the "if" clause intended in this place

Reply:
yes. 
This correspond to the case 
> Early access mode on + legacy CID off: no restriction

we discussed with @bame-da above.